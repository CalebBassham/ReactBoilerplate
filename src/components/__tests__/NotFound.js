import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../NotFound';

describe('NotFound Component (404)', () => {
  it('it should show 404 message', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
  })
})