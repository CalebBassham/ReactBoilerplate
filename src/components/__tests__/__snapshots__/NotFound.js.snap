// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Component (404) it should show 404 message 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFound />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": "404 - Page not found :(",
    },
    "ref": null,
    "rendered": "404 - Page not found :(",
    "type": "h1",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": "404 - Page not found :(",
      },
      "ref": null,
      "rendered": "404 - Page not found :(",
      "type": "h1",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
